person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :age, :name
+  attr_reader :id
+
+  def can_use_services?
+    return true if of_age? || Person.parent_permission
```

Comment:
- [ ] I also get undefined method here for `parent_permission`. Instead of `Person.parent_permission`, kindly use the `@parent_permission` instance variable. It is clear and concise and returns no errors.